docs/src/overview.md

```diff
@@ -52,13 +52,14 @@ different features on different mesh types.
 | Spatial dimension                                            |     1D, 2D, 3D     |        1D, 2D, 3D        |              2D              |        2D, 3D       |
 | Coordinates                                                  |      Cartesian     |        curvilinear       |          curvilinear         |     curvilinear     |
 | Connectivity                                                 |  *h*-nonconforming |        conforming        |          conforming          |  *h*-nonconforming  |
-| Element type                                                 | line, square, cube |    line, square, cube    |             square           | line, square, cube  |
+| Element type                                                 |  line, quadᵃ, hexᵃ |     line, quadᵃ, hexᵃ    |             quadᵃ            |  line, quadᵃ, hexᵃ  |
 | Adaptive mesh refinement ([`AMRCallback`](@ref))             |          ✅         |             ❌            |               ❌              |          ✅          |
 | Domain                                                       |      hypercube     |     mapped hypercube     |           arbitrary          |      arbitrary      |
```

Comment:
```suggestion
| Element type                                                 |line, squareᵃ, cubeᵃ|     line, quadᵃ, hexᵃ    |             quadᵃ            |     quadᵃ, hexᵃ     |
| Adaptive mesh refinement ([`AMRCallback`](@ref))             |          ✅        |             ❌            |               ❌             |          ✅          |
| Domain                                                       |      hypercube     |     mapped hypercube     |           arbitrary          |      arbitrary      |
```
I'd use square/cube for the TreeMesh to emphasize it's not arbitrary quadrilaterals/hexahedrons. Also, `P4estMesh` does not support 1D.